leasing/fixtures/service_unit.json

```diff
@@ -4,6 +4,7 @@
         "pk": 1,
         "fields": {
             "name": "Maaomaisuuden kehittäminen ja tontit",
+            "abbreviation": "MAKE / Tontit",
```

Comment:
In ticket there were no spaces around `/`.

Reply:
I put them for aesthetic reasons.

Reply:
But I looked at it again, and decided to take them out, to make it shorter.

Reply:
I added a comment on the ticket regarding the naming practice. I find it odd that it is not logical, but rather random